Tools/Sources/OutdatedPackages.swift

```diff
@@ -0,0 +1,23 @@
+import ArgumentParser
+import Foundation
+
+struct OutdatedPackages: ParsableCommand {
+    static var configuration = CommandConfiguration(abstract: "A tool to check outdated package dependencies, requires to have setup the project first")
```

Comment:
Maybe a little bit clearer:
```suggestion
    static var configuration = CommandConfiguration(abstract: "A tool to check outdated package dependencies. Please make sure you have already run setup-project before using this tool.")
```